projects/elife.yaml

```diff
@@ -2047,17 +2047,18 @@ data-pipeline:
                 tables:
                     # bit of repetition here
                     380_datascience_early_career_researchers:
-                        schema: https://raw.githubusercontent.com/elifesciences/data-pipeline-ejp-csv-deposit/master/schemas/380_datascience_early_career_researchers.json
+                        schema: ./projects/data-pipeline/bigquery/schemas/380_datascience_early_career_researchers.json
                     455_datascience_editors:
-                        schema: https://raw.githubusercontent.com/elifesciences/data-pipeline-ejp-csv-deposit/master/schemas/455_datascience_editors.json
+                        schema: ./projects/data-pipeline/bigquery/schemas/455_datascience_editors.json
                     489_datascience_editor_keywords:
-                        schema: https://raw.githubusercontent.com/elifesciences/data-pipeline-ejp-csv-deposit/master/schemas/489_datascience_editor_keywords.json
+                        schema: ./projects/data-pipeline/bigquery/schemas/489_datascience_editor_keywords.json
                     705_datascience_reviewer_identity_revealed_last_week:
-                        schema: https://raw.githubusercontent.com/elifesciences/data-pipeline-ejp-csv-deposit/master/schemas/705_datascience_reviewer_identity_revealed_last_week.json
+                        schema: ./projects/data-pipeline/bigquery/schemas/705_datascience_reviewer_identity_revealed_last_week.json
                     767_datascience_person_roles:
-                        schema: https://raw.githubusercontent.com/elifesciences/data-pipeline-ejp-csv-deposit/master/schemas/767_datascience_person_roles.json
+                        schema: ./projects/data-pipeline/bigquery/schemas/767_datascience_person_roles.json
                     779_datascience_reviewer_info_all:
-                        schema: https://raw.githubusercontent.com/elifesciences/data-pipeline-ejp-csv-deposit/master/schemas/779_datascience_reviewer_info_all.json
+                        schema: ./projects/data-pipeline/bigquery/schemas/779_datascience_reviewer_info_all.json
+                    # these are private
                     795_datascience_person_merge_info_all:
```

Comment:
I merge `master` because new tables were added recently. Please check if their schemas need to be brought in too.

Reply:
will do